NuRadioMC/EvtGen/generator.py

```diff
@@ -731,53 +799,7 @@ def generate_surface_muons(filename, n_events, Emin, Emax,
-        * fiducial_zmin: float
-            lower z coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-        * fiducial_zmax: float
-            upper z coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-        * full_rmin: float (optional)
-            lower r coordinate of simulated volume (if not set it is set to the fiducial volume)
-        * full_rmax: float (optional)
-            upper r coordinate of simulated volume (if not set it is set to the fiducial volume)
-        * full_zmin: float (optional)
-            lower z coordinate of simulated volume (if not set it is set to the fiducial volume)
-        * full_zmax: float (optional)
-            upper z coordinate of simulated volume (if not set it is set to the fiducial volume)
-
-        or a cube specified with
-
-        * fiducial_xmin: float
-            lower x coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-        * fiducial_xmax: float
-            upper x coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-        * fiducial_ymin: float
-            lower y coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-        * fiducial_ymax: float
-            upper y coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-        * fiducial_zmin: float
-            lower z coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-        * fiducial_zmax: float
-            upper z coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-        * full_xmin: float (optional)
-            lower x coordinate of simulated volume (if not set it is set to the fiducial volume)
-        * full_xmax: float (optional)
-            upper x coordinate of simulated volume (if not set it is set to the fiducial volume)
-        * full_ymin: float (optional)
-            lower y coordinate of simulated volume (if not set it is set to the fiducial volume)
-        * full_ymax: float (optional)
-            upper y coordinate of simulated volume (if not set it is set to the fiducial volume)
-        * full_zmin: float (optional)
-            lower z coordinate of simulated volume (if not set it is set to the fiducial volume)
-        * full_zmax: float (optional)
-            upper z coordinate of simulated volume (if not set it is set to the fiducial volume)
-
+        a dictionary specifying the simulation volume. See set_volume_attributes() for an explanation
```

Comment:
Please leave the definition of volume here. It is cumbersome if you need to keep jumping to additional functions to find the definitions. With a good IDE, the documentation is directly shown by hovering over the function. But now, the user would need to jump into this file and manually search for `set_volume_attributes`